.github/workflows/main.yml

```diff
@@ -65,14 +66,16 @@ jobs:
             make distclean ENABLE_GDBSTUB=1 gdbstub-test
     - name: JIT test
       run: |
-            make clean && make ENABLE_JIT=1 check -j$(nproc)
-            make clean && make ENABLE_EXT_A=0 ENABLE_JIT=1 check -j$(nproc)
-            make clean && make ENABLE_EXT_F=0 ENABLE_JIT=1 check -j$(nproc)
-            make clean && make ENABLE_EXT_C=0 ENABLE_JIT=1 check -j$(nproc)
+            export VER=-14
```

Comment:
Why `VER` here?

Reply:
Because the cloud machine can not find the `llvm-config` in CI, but `llvm-config-14` can be found.

Reply:
Can you support [recent LLVM releases](https://releases.llvm.org/)? e.g., LLVM 18.x